assets/.eslintrc.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 module.exports = {
-    extends: ['eslint:recommended', 'plugin:vue/recommended', 'plugin:compat/recommended'],
+    extends: ['eslint:recommended', 'plugin:vue/strongly-recommended', 'plugin:compat/recommended'],
```

Comment:
💪

Reply:
Weirdly "strongly-recommended" is weaker than "recommended" 🤷‍♂️